package.json

```diff
@@ -15,10 +15,11 @@
     "test": "NODE_ENV=test jest",
     "test:coverage": "yarn run test -- --coverage",
     "test:watch": "yarn run test -- --watch",
-    "lint": "yarn run lint:js && yarn run lint:css",
+    "lint": "yarn run lint:js && yarn run lint:css && yarn run lint:prettier",
```

Comment:
isn't this already done in `lint:js` with eslint-plugin-prettier? could probably remove that then?

Reply:
lol 2slow, yeah, just remove eslint-plugin-prettier :)